extensions/blocks/rating-star/rating-meta.php

```diff
@@ -61,6 +61,26 @@ function jetpack_rating_meta_get_symbol_high_fidelity( $attributes, $pos ) {
 	}
 }
 
+if ( ! function_exists( 'jetpack_rating_get_schema_for_symbol' ) ) {
```

Comment:
Do we need the additional function_exists check here? Is it used somewhere else already?

Reply:
All functions in this file followed that pattern so I stuck with it.

Reply:
Looking for more feedback here if required

Reply:
👍 Let's be consistent in this file then. It was most likely done to remain compatible with the old Star rating block that previously existed on WordPress.com.